utils/dataloaders.py

```diff
@@ -29,11 +29,11 @@ def get_dataloaders(batch_size, dataset):
         "celeba": get_celeba_dataloader
     }
     if dataset in dataset_options:
-        return dataset_options[dataset](batch_size=batch_size)
+        return dataset_options[dataset](batch_size=batch_size, shuffle=shuffle)
```

Comment:
Is it not neater to just call shuffle here rather than shuffle=shuffle?

Reply:
I like to be explicit about which parameter is being called, but I don't mind changing it if it is prefered @YannDubs .

Reply:
Happy to leave it if you would rather be explicit about it.

Reply:
I normally always give the argument key and value (i.e `shuffle=shuffle`) if it's an optional argument (to avoid possible bugs  / not have to remember the order in which I wrote the function / works on all python version which is not the case if you don't) but never for required arguments (to be more concise). I.e. I would not for the `batch_size` in this case.